src/ProviderPR/SFA.DAS.ProviderPR/Application/Queries/GetRelationshipByEmail/GetRelationshipByEmailQueryResult.cs

```diff
@@ -0,0 +1,40 @@
+﻿using SFA.DAS.SharedOuterApi.Models.ProviderRelationships;
+
+namespace SFA.DAS.ProviderPR.Application.Queries.GetRelationshipByEmail;
+
+public record GetRelationshipByEmailQueryResult
+{
+    public bool HasUserAccount { get; init; }
+    public bool? HasOneEmployerAccount { get; init; }
+    public long? AccountId { get; init; }
+    public bool? HasOneLegalEntity { get; init; }
+    public string? AccountLegalEntityPublicHashedId { get; init; }
+    public long? AccountLegalEntityId { get; init; }
+    public string? AccountLegalEntityName { get; init; }
+    public bool? HasRelationship { get; init; }
+    public List<Operation>? Operations { get; init; } = new List<Operation>();
+
+    public GetRelationshipByEmailQueryResult(bool hasUserAccount, bool? hasOneEmployerAccount, long? accountId,
```

Comment:
As suggested in the previous comment, if the properties are known to be defaulted to null then I would suggest having them set as null on the constructor allowing for the creation of the object without the need to set them explicitly.